apps/web/src/features/resources/layout.tsx

```diff
@@ -30,14 +33,15 @@ export const ResourceLayout = ({
   pagination,
   className,
   link,
+  categoryActive,
 }: Props) => {
   return (
     <MainLayout footerVariant="dark">
-      <div
-        className={`flex h-fit min-h-screen justify-center p-2 sm:p-10 ${className}`}
-      >
-        <div className="max-w-6xl text-black">
-          <div>
+      <div className={`flex h-fit justify-center p-2 md:p-10 ${className}`}>
```

Comment:
Use the `cn` utility from our ui package to merge classes. Only commenting here but I've seen other occurrences.